etl/embedding_model_pipelines/open_ai_embedding_model_pipeline.py

```diff
@@ -0,0 +1,47 @@
+from langchain.docstore.document import Document
+from langchain.embeddings import CacheBackedEmbeddings
+from langchain.storage import LocalFileStore
+from langchain_community.vectorstores import FAISS
+from langchain_core.embeddings import Embeddings
+from langchain_core.vectorstores import VectorStore
+from langchain_openai import OpenAIEmbeddings
+
+from etl.embedding_model_pipelines import EmbeddingModelPipeline
+from etl.models import OpenAiSettings, OutputConfig
+
+
+class OpenAiEmbeddingModelPipeline(EmbeddingModelPipeline):
+    """
+    A concrete implementation of EmbeddingModelPipeline.
+
+    Uses OpenAI's embedding models to transform Records into embeddings.
+    """
+
+    def __init__(
+        self, *, openai_settings: OpenAiSettings, output_config: OutputConfig
+    ) -> None:
+        self.__openai_settings: OpenAiSettings = openai_settings
+        self.__parsed_output_config: OutputConfig.Parsed = output_config.parse()
+
+    def __create_embedding_model(self) -> Embeddings:
+        """Create and return an OpenAI embedding model."""
+
+        self.__parsed_output_config.directory_path.mkdir(exist_ok=True)
```

Comment:
Should probably put the embeddings in their own subdirectory. See note above re: computed directory paths hanging off `Parsed`.

Reply:
The embeddings now go in the  `/output/openai_embeddings_cache` subdirectory.